src/sentry/integrations/metric_alerts.py

```diff
@@ -74,7 +74,9 @@ def incident_attachment_info(incident, metric_value=None, action=None, method=No
     else:
         metric_and_agg_text = f"{metric_value} {agg_text}"
 
-    text = f"{metric_and_agg_text} in the last {time_window} minutes"
+    interval = "minute" if time_window == 1 else "minutes"
```

Comment:
🤔 should we be using the translation functions on these things?

Reply:
Is that this `ugettext_lazy` ?

Reply:
Yeah, there's a `_` version of it too